src/lockfiles.nim

```diff
@@ -133,113 +131,92 @@ proc write(lock: NimbleLockFile; lockFilePath: string) =
 
 proc genLockEntry(c: var AtlasContext;
                   lf: var NimbleLockFile;
-                  pkg: Package,
+                  w: Dependency,
                   cfg: CfgPath,
-                  deps: HashSet[PackageName]) =
-  let info = extractRequiresInfo(pkg.nimble.string)
+                  deps: HashSet[string]) =
```

Comment:
> deps: HashSet[string])

Do we use the resolved repo name here? The local repo name, or the last part of the url one? :)